run-smoke-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+cd test/fixtures && python -m SimpleHTTPServer 9999 &
+
+./cli.js http://localhost:9999/online-only.html > results
+
+if ! grep -q "URL responds with a 200 when offline: false" results; then
```

Comment:
:+1: started doing a version using jq . to parse the response but since it is just an array with objects and things are not keyed inside of an object, the matching logic got much worse and uglier than this.

:+1: to iterating more